app/env.py

```diff
@@ -46,6 +47,7 @@
 OSF_BASE_URL = os.environ.get("OSF_BASE_URL", "https://osf.example")
 OSF_API_BASE_URL = os.environ.get("OSF_API_BASE_URL", "https://api.osf.example")
 OSF_AUTH_COOKIE_NAME = os.environ.get("OSF_AUTH_COOKIE_NAME", "osf_staging")
+OSF_AUTH_COOKIE_SECRET = os.environ.get("OSF_AUTH_COOKIE_SECRET", "CHANGEME")
```

Comment:
GV Deployment Note:

* `OSF_AUTH_COOKIE_NAME` should be the same as what OSF uses in `COOKIE_NAME`
* `OSF_AUTH_COOKIE_SECRET` should be the same as what OSF uses in `SECRET_KEY`